traffic_monitor/cache/stats_over_http.json

```diff
@@ -1,5 +1,6 @@
 {
 	"global": {
+		"plugin.system_stats.timestamp_ms": "1684784877939",
```

Comment:
Does the `current_time_epoch_ms` field still need to be in this file?

Reply:
Doesn't hurt.. that was also what got added when the other (plugin field) got added